src/cascadia/PublicTerminalCore/framework.h

```diff
@@ -0,0 +1,5 @@
+#pragma once
+
+#define WIN32_LEAN_AND_MEAN             // Exclude rarely-used stuff from Windows headers

```

Comment:
Why isn't whomever is referencing this file just using `LibraryIncludes.h` or why isn't this just in PCH?